README.md

```diff
@@ -1,6 +1,97 @@
 # Medical Event Data Standard
 
-The Medical Event Data Standard (MEDS) is a draft data schema for storing streams of medical events, often sourced from either Electronic Health Records or claims records.
+The Medical Event Data Standard (MEDS) is a data schema for storing streams of medical events, often
+sourced from either Electronic Health Records or claims records. Before we define the various schema that make
+up MEDS, we will define some key terminology that we use in this standard.
+
+## Terminology
+  1. A _patient_ in a MEDS dataset is the primary entity being described by the sequences of care observations
+     in the underlying dataset. In most cases, _patients_ will, naturally, be individuals, and the sequences
+     of care observations will cover all known observations about those individuals in a source health
+     datasets. However, in some cases, data may be organized so that we cannot describe all the data for an
+     individual reliably in a dataset, but instead can only describe subsequences of an individual's data,
+     such as in datasets that only link an individual's data observations together if they are within the same
+     hospital admission, regardless of how many admissions that individual has in the dataset (such as the
+     [eICU](https://eicu-crd.mit.edu/) dataset). In these cases, a _patient_ in the MEDS dataset may refer to
+     a hospital admission rather than an individual.
+  2. A _measurement_ or _patient measurement_ or _observation_ in a MEDS dataset refers to a single measurable
+     quantity observed about the patient during their care. These observations can take on many forms, such as
+     observing a diagnostic code being applied to the patient, observing a patient's admission or transfer
+     from one unit to another, observing a laboratory test result, but always correspond to a single
```

Comment:
I don't think the measurement abstraction exists in MEDS anymore since we don't have nesting.

These are just "events" now. "event" is now the term for a single measurable quantity observed about a patient during their care.

This section should be rewritten to refer to "event"

Reply:
I don't have an objection to using "event" instead of "measurement", but I want there to be some term to refer to all observations that occur at a single unique timestamp. This is useful in several contexts, even when we don't nest, such as:
1. Providing a per-patient integral index of the ordered unique timestamps in their sequence that can be used during tabularization (meds tab uses this), for matching to label cohorts (meds-tab and meds-torch use this), or efficiently subselecting tensorized data over a certain time range (the nested ragged tensorization code uses this).
2. Differentiating pre-processing transformations that affect the observations within a timepoint but do not affect the timepoints of the data from those that do affect the timepoints of the data.
3. Differentiating models that operate over sequences of observations/measurements vs. those that operate over sequences of unique timestamps.

To me, the natural division is to use the term "event" to occur to all things at a unique timestamp and "measurement" or "observation" to refer to a single tuple of `patient_id`, `timestamp`, `code`, and `*_value`, but I'm open to using different verbiage, as long as we can express both concepts. Do you have an alternate nomenclature you'd prefer?

Reply:
I don't think MEDS needs to define that term, or use it, since the schema/datamodel doesn't require referring to unique timestamps.

I'm not sure what I would call this abstraction since I have never used it (and don't really see a case where I would).